### PR TITLE
Check rrule license header

### PR DESCRIPTION
**Pull request by Yarovaya**

### CODE REVIEW

**Review by AntonSermyazhko — DISMISSED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ matrix:
   include:
     - { TARGET: lint }
     - { TARGET: ts }
+    - { TARGET: test_license }
```
Comment (San4es) [inline .drone.yml]: We have similar configs for Shippable and Travis.

File: docker-ci.sh [hunk 2]
```diff
@@ -195,6 +195,13 @@ function run_test_scss {
     npm i && npm run build && npm run test
 }
 
+function run_test_license {
+    npm i
+    npm run build
+
+    npx gulp check-license-notices
```
Comment (San4es) [inline docker-ci.sh]: This task is called twice. See:
https://devextreme-ci.devexpress.com/DevExpress/DevExtreme/34035/51

Comment (San4es) [inline docker-ci.sh:202]: This task is called twice: https://devextreme-ci.devexpress.com/DevExpress/DevExtreme/34035/51

File: docker-ci.sh [hunk 3]
```diff
@@ -195,6 +195,13 @@ function run_test_scss {
     npm i && npm run build && npm run test
 }
 
+function run_test_license {
```
Comment (San4es) [inline docker-ci.sh]: I have doubts about creating a separate task in the CI matrix:
- it builds everything but we check only one file (dx.all.js).
- `npm run build` takes about 8 minutes: https://devextreme-ci.devexpress.com/DevExpress/DevExtreme/34035/51
- the `check-license-notices` task takes only 2 sec.

Looks like we can run the `check-license-notices` task in Gulp tasks where we build any bundles (dx.all, dx.all.debug) with and without uglify.

Comment (AntonSermyazhko) [reply docker-ci.sh]: If we check the license in the debug version, there is no guaranty that the license header will be presented in the product one. The product bundle is not build by any task of the tests matrix, so we did a separate task to check the license. And in this case there is no excessive consumption of process time.

File: docker-ci.sh (hunk 2 above)
Comment (Yarovaya) [reply docker-ci.sh]: I've updated PR and then we call 'check-license-notices' task only in 'test_license' task

File: docker-ci.sh (hunk 3 above)
Comment (Yarovaya) [reply docker-ci.sh]: Important notice - the 'check-license-notices' task takes 30 minutes for dx.all.debug.js.

**Review by AntonSermyazhko — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Pull request merged**